packages/python/accern_data/accern_data.py

```diff
@@ -553,43 +546,30 @@ def get_last_silenced_errors(
             self) -> List[Tuple[Optional[Dict[str, str]], str]]:
         return list(self._error_list)
 
-    @staticmethod
-    def _get_date_type(obj: Dict[str, str]) -> Tuple[str, str]:
-        for date_type in ["date", "harvested_at"]:
-            try:
-                date_val = obj[date_type]
-                break
-            except KeyError:
-                pass
-        if date_type == "date":
-            date_type = "published_at"
-        return date_type, date_val
-
     def _read_total(
             self,
-            date: Dict[str, str],
-            filters: Dict[str, str],
+            params: Dict[str, str],
```

Comment:
what does params contain? at this point it should only be url_params -- all other params should go to json